resources/js/App/Components/Header/Header.jsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+
+import "./header.css";
+import { Link } from "react-router-dom";
+
+
+class Button extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    return(
+      <Link to={`/`}>
+        <button onClick={this.props.changeThePage}>
```

Comment:
Why do you need both `Link` and `button` with `onClick`? It seems like a mistake.

I would only use Link here and handle the logic through React Router.